src/actions/auth.js

```diff
@@ -1,4 +1,7 @@
 import { currentBackend } from '../backends/backend';
+import { actions as notifActions } from 'redux-notifications';
+
+const { notifSend } = notifActions;
```

Comment:
Today I learned you can't do full destructuring in an `import` call - I was going to suggest `import { actions: { notifSend } } from 'redux-notifications';`, but that syntax is only supported for assignment, not imports).